tests/forward_master/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
In order to run the newly added case in CI, you should add it to _tests/other_integration.txt_.